rpc/serializers.py

```diff
@@ -378,30 +378,61 @@ def get_slug(self, obj) -> str:
 
 @dataclass
 class SubseriesDoc:
-    type: str
-    number: int
-    members: list[int]
+    name: str
+    rfc_number: int
 
-    @property
-    def rfc_count(self):
-        return len(self.members)
 
-    @property
-    def display_name(self):
-        return f"{self.type.upper()} {self.number}"
+class SubseriesDocSerializer(serializers.Serializer):
+    name = serializers.CharField()
+    rfc_number = serializers.IntegerField()
 
-    @property
-    def slug(self):
-        return f"{self.type.lower()}{self.number}"
 
+@dataclass
+class Subseries:
+    type: str
+    number: int
 
-class SubseriesDocSerializer(serializers.Serializer):
+
+class SubseriesSerializer(serializers.Serializer):
     type = serializers.CharField()
     number = serializers.IntegerField()
-    members = serializers.ListField(child=serializers.IntegerField())
-    rfc_count = serializers.IntegerField(read_only=True)
-    slug = serializers.CharField(read_only=True)
-    display_name = serializers.CharField(read_only=True)
+    documents = serializers.SerializerMethodField()
+    rfc_count = serializers.SerializerMethodField()
+    slug = serializers.SerializerMethodField()
+    display_name = serializers.SerializerMethodField()
+
+    @extend_schema_field(SubseriesDocSerializer(many=True))
+    def get_documents(self, obj):
+        type_slug = obj.type
+        number = obj.number
+
+        documents = SubseriesMember.objects.select_related("rfc_to_be__draft").filter(
+            type__slug=type_slug, number=number
+        )
+        docs = []
+        for doc in documents:
+            rfc_to_be = doc.rfc_to_be
+            draft = getattr(rfc_to_be, "draft", None) if rfc_to_be else None
+            if rfc_to_be and draft:
+                docs.append(
+                    {
+                        "name": draft.name,
+                        "rfc_number": rfc_to_be.rfc_number,
+                    }
+                )
```

Comment:
After refactoring this method into a `@property` on the dataclass, it should just return the document queryset (or list, or whatever iterable is convenient) here.